solutions/silver/cf-1744F.mdx

```diff
@@ -0,0 +1,144 @@
+---
+id: cf-1744F
+source: CF
+title: MEX vs MED
+author: Alex Du
+---
+
+## Explanation
+
+[Official Analysis (No Code)](https://codeforces.com/blog/entry/108101)
+
+<Spoiler title="Hint 1">
+
+How do we know if a specific subsegment satisfies the constraints for a given
+$\texttt{mex}$?
+
+First, to reach that value of $\texttt{mex}$, we must find subsegments which contain all
+integers $0, 1, \ldots, \texttt{mex}-1$, while also not containing $\texttt{mex}$.
+
+Then, given a subsegment, $\texttt{mex} > \texttt{med}$ will be satisfied if the length of the
+subsegment is $\leq 2 \cdot \texttt{mex}$, since the median will be at most $\texttt{mex}-1$.
```

Comment:
```suggestion
For a subsegment of length $\leq 2 \cdot \texttt{mex}$, the condition$\texttt{mex} > \texttt{med}$ will be satisfied, since the median can be at most $\texttt{mex}-1$.
```
Add this in the separate hint as @SansPapyrus683 mentioned